src/main/java/com/microsoft/azure/relay/HybridConnectionListener.java

```diff
@@ -111,6 +113,57 @@ public void setMaxWebSocketBufferSize(int maxWebSocketBufferSize) {
 		}
 	}
 
+	/**
+	 * Returns the handler that will be run when the listener disconnects unexpectedly due to potential error.
```

Comment:
>due to potential error [](start = 84, length = 22)

The "due to potential error" doesn't seem needed to me. If the listener disconnects unexpectedly it's an error in my mind.